lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -1,56 +1,77 @@
+$panel-properties-height: 56px !default;
+
 .adf {
     &-metadata-properties {
-        .mat-expansion-panel-header.mat-expanded:hover,
-        .mat-expansion-panel-header.mat-expanded:focus {
-            background: var(--adf-theme-background-hover-color);
+        .mat-expansion-panel:not([class*='mat-elevation-z']) {
```

Comment:
Can we avoid changing internal mat classes?

Reply:
I can try again, already tried few times but there's a corner case clashing with other panel styles, will investigate